### DISCUSSION TITLE
请问该如何设置并改变网站中显示的字体呢，想使用自己的ttf字体

### DISCUSSION THREAD

**Discussion by SolEtStella** [hugo-next/hugo-theme-next#150] [Q&A]
新手小白入门想使用主题建站，期待大佬答疑解惑

**Comment by elkan1788** [comment]
@Kazan-oc 这个功能暂时还没完成开发，可以尝试找下如何加载网络路径的字体库，然后在主题的对应的代码位置中（[#L89C1-L91C62](https://github.com/hugo-next/hugo-theme-next/blob/67b5ae3a3ab3d5da32121647e966548f446b290e/assets/css/_variables/base.scss#L89C1-L91C62)）修改自己的字体库链接地址。
